package.json

```diff
@@ -35,14 +37,16 @@
   ],
   "scripts": {
     "test": "istanbul cover node_modules/mocha/bin/_mocha -- -R spec --require should",
+    "prepare": "npm run build",
     "build": "npm-run-all --sequential build:*",
-    "build:rimraf": "rimraf ./build",
-    "build:mkdirp": "mkdirp ./build",
+    "build:rimraf": "rimraf ./lib && rimraf ./dist",
+    "build:mkdirp": "mkdirp ./lib && mkdirp ./dist",
     "build:dedupe": "npm dedupe",
-    "build:browserify": "browserify index.js -o ./build/tmi.js",
-    "build:uglify": "uglifyjs --compress --mangle --output ./build/tmi.min.js --source-map ./build/tmi.js.map ./build/tmi.js",
-    "build:sri": "node sri.js ./build/tmi.js",
-    "build:sri-min": "node sri.js ./build/tmi.min.js"
+    "build:lib": "babel ./src --out-dir ./lib --presets=es2015",
+    "build:browserify": "browserify ./lib/index.js -o ./dist/tmi.js",
```

Comment:
Hehe yes ... reluctantly, I changed `build` to `dist`. I was following the pattern used in other popular libraries.